doc/man3/X509_STORE_CTX_get_error.pod

```diff
@@ -228,6 +228,7 @@ The root CA is marked to reject the specified purpose.
 
 The current candidate issuer certificate was rejected because its subject name
 did not match the issuer name of the current certificate.
+Not used as of OpenSSL 1.1.0.
```

Comment:
This should be in a history section.  However, 1.1.0 is unsupported so it might be better to just leave this bit out.

Reply:
The same note is in 3 other error code descriptions in that file and all these 4 codes have the same description in 1.1.1 [verify.pod](https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/doc/man1/verify.pod). As I see it, these codes are unused since 1.1.0, not only in 1.1.0.